package.json

```diff
@@ -64,10 +65,8 @@
   },
   "scripts": {
     "prepare": "node scripts/copyFiles",
-    "build-sass": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
-    "watch-sass": "npm run build-sass && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
-    "start": "npm run prepare && npm run build-sass && react-scripts start",
-    "build": "npm run prepare && npm run build-sass && react-scripts build",
+    "start": "npm run prepare && react-scripts start",
```

Comment:
Are you sure the static .css file is not used anymore ?

Reply:
yes, https://github.com/bitfinexcom/bfx-report-ui/pull/245/files#diff-1fdf421c05c1140f6d71444ea2b27638R19

Reply:
okay, just make sure the static files is not used for bundle injecting for dev mode. double check with someone that usually dev on this.